### PR TITLE
feat(OHIFv3):Add static e2e data

### PR DESCRIPTION
**Pull request by wayfarer3130**
Make e2e tests run against static data served up by the standard webpack process.

### CODE REVIEW

File: extensions/default/src/DicomWebDataSource/utils/StaticWadoClient.js [hunk 1]
```diff
@@ -15,6 +15,7 @@ export default class StaticWadoClient extends api.DICOMwebClient {
     "StudyDescription": "00081030",
     "StudyDate": "00080020",
     "ModalitiesInStudy": "00080061",
+    AccessionNumber: "00080050",
```
Comment (wayfarer3130) [inline extensions/default/src/DicomWebDataSource/utils/StaticWadoClient.js:18]: Need to list all the query fields in order to pass the e2e tests.  This fixes 1 test.

File: extensions/default/src/DicomWebDataSource/utils/StaticWadoClient.js [hunk 2]
```diff
@@ -66,7 +67,16 @@ export default class StaticWadoClient extends api.DICOMwebClient {
       actual = actual.Alphabetic;
     }
     if (typeof (actual) == 'string') {
-      return actual.indexOf(desired) != -1;
+      if (actual.length === 0) return true;
```
Comment (wayfarer3130) [inline extensions/default/src/DicomWebDataSource/utils/StaticWadoClient.js:70]: Compare with wild cards - not strictly needed for the e2e tests, but I prefer to keep things close to original.

File: package.json [hunk 3]
```diff
@@ -107,17 +107,17 @@
     "prettier": "^1.18.2",
     "react-hot-loader": "^4.13.0",
     "serve": "^11.1.0",
+    "shader-loader": "^1.3.1",
     "start-server-and-test": "^1.10.0",
     "style-loader": "^1.0.0",
     "terser-webpack-plugin": "^5.1.4",
     "unused-webpack-plugin": "2.4.0",
+    "webpack": "^5.50.0",
```
Comment (wayfarer3130) [inline package.json:115]: Need a newer webpack for static website data.

File: platform/viewer/public/config/local_static.js [hunk 4]
```diff
@@ -12,16 +12,16 @@ window.config = {
       sourceName: 'dicomweb',
       configuration: {
         name: 'DCM4CHEE',
-        wadoUriRoot: 'http://localhost:5000',
-        qidoRoot: 'http://localhost:5000',
-        wadoRoot: 'http://localhost:5000',
+        wadoUriRoot: '/dicomweb',
```
Comment (wayfarer3130) [inline platform/viewer/public/config/local_static.js:15]: Also use the new webpack for serving up ~/dicomweb by default.

**Comment by wayfarer3130** [comment]
@swederik - how do I make the CI builds also use the static data?  
I'm reconsidering where to store the data - I think it should be stored direct in viewer-testdata as an external git include from standard OHIF, already converted to dicomweb.

**Review by swederik — APPROVED** [review]

**Pull request merged**